website/Dockerfile

```diff
@@ -24,4 +24,4 @@ EXPOSE 3000
 VOLUME /config
 VOLUME /log
 
-CMD node ./dist/server/entry.mjs
+CMD node --max-old-space-size=1024 ./dist/server/entry.mjs
```

Comment:
We may want the kubernetes limit to never be smaller than this, ideally we derive this here from the kubernetes limit (passing in a runtime through env variable)